### ISSUE TITLE
Where to put focus after returning the molecule?

### ISSUE THREAD

**Issue by terracoda** [phetsims/molecules-and-light#230]
When we were testing [dev version 1.5.0-dev.18](https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-dev.18/phet/molecules-and-light_en_phet.html?supportsSound&a11y) we realized that moving focus to the light source slider after finding and pressing the "Return Molecule" button may not necessarily the best place to be because the molecule keeps breaking apart (edited).

**Question:**
Where to put focus when the molecule is returned after a break-apart and the learner finds and presses the Return Molecule button?

1. First thought was that focus should be returned to the previously focused item before the break apart happened, but that could be quite a few steps back depending on how long it takes for the learner to find the "Return Molecule" button. 
2. Second thought: Since both the molecule radios and the "Return Molecule" button can return a molecule, @Matthew-phet and @terracoda are wondering if moving focus to the returned molecule radio button would better.

**Benefits & Reasoning:**
This location has the benefit of communicating to the user which molecule gets returned. In addition, it would be easy to return a molecule accidentally (or intentionally) by selecting a new molecule with an Arrow up or Arrow down interaction.

@jessegreenberg, is it possible adjust the focus management so that when the Return Molecule button disappears (after being activating) the focus goes to the radiobutton of the returned molecule? This would be one of two molecules:
 - Nitrogen Dioxide, or
 - Ozone

whichever one is being returned.

If straight forward, please implement so we can have this feature for interview sessions (Dec 18th & 19th).

If not straight forward, please add your thoughts.

**Comment by jessegreenberg** [comment]
I investigated and unfortunately this will be difficult. The individual radio buttons are private to the group and so I cannot (with the current implementation of RadioButtonGroup) set focus programmatically to one button.

As a sighted user I think it makes most intuitive sense to put either focus on the slider (since it is also in the observation window), or to just make focus go nowhere and have the highlight disappear.

But another option could be to move focus to the last item that had focus before the button. If tabbing forward to the "Return Molecule" button, focus will move back to the slider on press. If shift-tabbing backwards to the "Return Molecule" button, focus moves back to the molecule radio button on press.

Back to @terracoda  and @Matthew-phet

**Comment by terracoda** [comment]
Ok, @jessegreenberg, let's leave it as is - on the light source slider... and see what blind learners make of it. 

Thanks for investigating. We can re-visit/re-open after if needed after interviews.
